ci/it/testcases/test_reading_clickhouse_tables.go

```diff
@@ -62,5 +62,5 @@ func (a *ReadingClickHouseTablesIntegrationTestcase) testRandomThing(ctx context
 	}
 	defer resp.Body.Close()
 	assert.Equal(t, "Clickhouse", resp.Header.Get("X-Quesma-Source"))
-	assert.Equal(t, 200, resp.StatusCode) // TODO INTENTIONAL FAILURE
+	//assert.Equal(t, 200, resp.StatusCode) // TODO INTENTIONAL FAILURE
```

Comment:
```suggestion
```

delete it instead of commenting?

Reply:
https://github.com/QuesmaOrg/quesma/pull/875/commits/31e4d4757c438bac5974b031399e2259f49a418c